scope_ascom.cpp

```diff
@@ -136,6 +136,12 @@ wxArrayString ScopeASCOM::EnumAscomScopes()
     return list;
 }
 
+static bool AlreadyPresentInGit(GITEntry deviceEntry)
+{
+    IDispatch *idisp = deviceEntry.Get();
+    return idisp != nullptr;

```

Comment:
Get() [adds a reference](https://learn.microsoft.com/en-us/windows/win32/api/objidl/nf-objidl-iglobalinterfacetable-getinterfacefromglobal#remarks) to the object so as written this leaks a reference.
The GITObjRef class can be used for this purpose:
```suggestion
    return GITObjRef(deviceEntry).IDisp() != nullptr;
```